tests/unit/test_session.py

```diff
@@ -0,0 +1,523 @@
+from .helpers import TestHelpers
+from aiohttp.test_utils import unittest_run_loop
+
+
+class TestSessionHandling(TestHelpers):
```

Comment:
we have missed an opportunity to add code coverage to the security "invalidate" method. Which is session relevant.

Reply:
The invalidate method was introduced as part of CR-1629-Exit-Improvement. That has unit tests delivered with it but I didn't review that stream. I rebased this stream to master several times during development picking up that work but that functionality was I believe out of scope of this PR? Happy to look at it if we raise another JIRA ticket?

Reply:
i understand, just wondered if we could go the extra mile while you were right in the code

Reply:
Think we should get this in ASAP and maybe this shouldn't hold up delivery. Can add unit tests to look at invalidation of the session as part of another JIRA but would want to spend time looking at what it actually does.